### PR TITLE
Integration tests : Edit Operator Flow

### PR DESCRIPTION
**Pull request by vishwajeetraj11**
- [x] Press the edit button in the right side of the cp url input before entering the input.
- [x] Add Operator Edit flow

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,8 @@
 PUBLIC_NODE_ENV="development"
 VITE_PROJECT_ID=""
 
-# for testing only
+# Testing
 VITE_SEED_PHRASE=""
-VITE_WALLET_PASSWORD=""
\ No newline at end of file
+VITE_WALLET_PASSWORD=""
```
Comment (nayanmarlin) [inline .env.example:6]: What is VITE_WALLET_PASSWORD here @vishwajeetraj11 ? is it wallet id?

**Review by nayanmarlin — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (vishwajeetraj11) [reply .env.example:6]: its wallet password. i.e. metamask wallet password that a user uses to unlock wallet.

**Review by nayanmarlin — APPROVED** [review]

File: src/lib/page-components/oyster/operator/OysterOperatorRegisterPage.svelte [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<script lang="ts">
+<script lang="ts" defer>
```
Comment (souvikmishra) [inline src/lib/page-components/oyster/operator/OysterOperatorRegisterPage.svelte]: why have we added defer here?

File: README.md [hunk 3]
```diff
@@ -37,6 +37,9 @@ You should see a prompt that looks like this:
 
 > The port number and VITE version can be different for you.
 
+## 🧪 Testing
+refer to test/docs.md
```
Comment (souvikmishra) [inline README.md:41]: maybe add in a relative link to the docs

File: src/lib/page-components/oyster/operator/OysterOperatorRegisterPage.svelte (hunk 2 above)
Comment (vishwajeetraj11) [reply src/lib/page-components/oyster/operator/OysterOperatorRegisterPage.svelte]: removed.

File: README.md (hunk 3 above)
Comment (vishwajeetraj11) [reply README.md]: added

**Review by souvikmishra — APPROVED** [review]

**Pull request merged**